### DISCUSSION TITLE
About binary upgrade

### DISCUSSION THREAD

**Discussion by mp075496706** [gogs/gogs#7079] [Q&A]
### What needs to be improved? Please describe

In the item "about binary upgrade", it is mentioned to copy the "custom", "data" and "log" folders to the gogs folder of the new version. However, after using for a long time, the size of these three folders will become very large. For example, I now have more than ten gigabytes.
So is it possible to copy all the files in the gogs folder of the new version to the old version for replacement? Will it take less time to upgrade the version?

### Why do you think it is important?

Upgrading the version requires simpler steps and less time-consuming.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by unknwon** [comment]
> So is it possible to copy all the files in the gogs folder of the new version to the old version for replacement?

Yeah, I think you can do the other way around, just move new files downloaded for the new version to the old directory.
